components/main_layout/header/view.coffee

```diff
@@ -97,6 +97,9 @@ module.exports = class HeaderView extends Backbone.View
     false
 
   openAuth: (options) ->
+    if sd.SHOW_ARTIST_CTA
```

Comment:
Similar to the follow button, if we are triggered from an artist page with the CTA loaded, we want to trigger the custom event (which is handled in the CTA view), and early return.